src/shared/components/annotation/CancerHotspots.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import Tooltip from 'rc-tooltip';
+import DefaultTooltip from 'shared/components/DefaultTooltip';
```

Comment:
oh, yeah. I did it because I was finding it annoying to guess/figure out how many ".." to apply before I was in the right directory haha. Do you want me to change it?